Bottom: ac6413e141fb717a618cb12aa2415c09a65291c0
Top:    87bbb4645c6813d59d20b53f5780ee1dc7dadf58
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-08 21:17:58 +0000

Refresh of thermal-c-add-warning-pointing

---

diff --git a/src/thermal.c b/src/thermal.c
index 445b97d..fa87f28 100644
--- a/src/thermal.c
+++ b/src/thermal.c
@@ -432,6 +432,7 @@ ThermPoly (PCBTypePtr p, PinTypePtr pin, Cardinal laynum)
         arc = CirclePoly (pin->X, pin->Y, pin->Thickness / 2);
         /* create a thin ring */
         poly_Boolean_free (pa, arc, &m, PBO_SUB);
+#warning THIS WOULD BE FASTER IF WE JUST SUTRACTED A CROSS SHAPE
         /* fix me needs error checking */
         if (style == 2)
           {
